src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="container">
+    <div class="calendar container__calendar calendar--month-length">
```

Comment:
you have to add modifier to switch the sterting position of the day in a week. imagine that your first day in not a monday. so if TUE i have ti start on the second column
![image](https://user-images.githubusercontent.com/31005050/194062457-b34db03d-70ab-4392-a561-29fea6e001a7.png)

Reply:
i made it throug function )
you need just to change $startingDay choose a day to start and  $daysAmount for the month lenght)

Reply:
Agree. Add here modifier `start-day` )

Reply:
thanks) excuse me